patterns/transactional-outbox/src/producer/lib/messages/fail.ts

```diff
@@ -0,0 +1,15 @@
+import {MessageRecord} from "./types";
+import {MessageModel} from "./model";
+
+export const recordFailure = async (message: MessageRecord) => {
+  const record = await MessageModel.findOne({
+    where: {id: message.id}
+  });
+
+  if(!record) throw new Error('pending message not found');
```

Comment:
This would not be desirable in production, yeah? Or am I missing something?

Reply:
Maybe Boom handles it?

Reply:
yeah boom normalizes the error into a standard shape for all responses, can't remember if it gives out revealing info